src/fsharp/vs/ServiceLexing.fsi

```diff
@@ -15,6 +15,9 @@ open System.Collections.Generic
 open Microsoft.FSharp.Compiler 
 open Microsoft.FSharp.Compiler.Range
 
+type Position = int * int
```

Comment:
Could we get comment precising if they differ from range in compiler itself: https://github.com/Microsoft/visualfsharp/blob/2ffb0b6c99a15b364af309dabb12504cc72ad3be/src/fsharp/range.fsi#L91

If they are supposed to conform to range.fsi, maybe reuse that type, if not it wouldn't hurt to state in a comment how the indices are considered in this area of the code.

This is an area where technical debt was accumulated in VFPT (where + 1 and - 1 are applied in various places without greatest consistency), we intend to standardize on FSharp.Compiler.Services types and convert the one from VS pretty close in the UI before involving our services implementing the logic in relation to FCS.